Gemfile.lock

```diff
@@ -107,28 +105,28 @@ GEM
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 3.0)
       netrc (~> 0.7)
-    sass (3.4.16)
-    sass-rails (5.0.3)
+    sass (3.4.18)
+    sass-rails (5.0.4)
       railties (>= 4.0.0, < 5.0)
       sass (~> 3.1)
       sprockets (>= 2.8, < 4.0)
       sprockets-rails (>= 2.0, < 4.0)
-      tilt (~> 1.1)
-    sprockets (3.2.0)
+      tilt (>= 1.1, < 3)
+    sprockets (3.3.4)
       rack (~> 1.0)
     sprockets-rails (2.3.2)
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       sprockets (>= 2.8, < 4.0)
-    stripe (1.23.0)
+    stripe (1.25.0)
       json (~> 1.8.1)
       rest-client (~> 1.4)
     thor (0.19.1)
     thread_safe (0.3.5)
-    tilt (1.4.1)
+    tilt (2.0.1)
```

Comment:
error may have something to do with all these gem version changes, especially tilt